temoa/extensions/modeling_to_generate_alternatives/tech_activity_vector_manager.py

```
@@ -107,22 +130,22 @@ def initialize(self) -> None:
             self.technology_size[tech] += 1
-            self.variable_index_mapping[tech][self.base_model.v_flow_out_annual.name].append(idx)
+            self.variable_index_mapping[tech][self.base_model.v_flow_out_annual.name].append(idx_annual)
         logger.debug('Catalogued %d Technology Variables', sum(self.technology_size.values()))
 
     @property
     def expired(self) -> bool:
         return False  # this Manager can always generate more...
 
-    def group_variable_names(self, tech) -> list[Var]:
-        return list(self.category_mapping.keys())
+    def group_variable_names(self, tech: str) -> list[str]:
```

Ruff (0.14.10)

Unused method argument: `tech`

(ARG002)